app/models/station.rb

```diff
@@ -60,40 +62,40 @@ def most_frequent_bike_id(trip, bike)
     Trip.where(bike_id: trip.bike_id).count
   end
 
-  def rides_started_at_station(trip)
```

Comment:
@maxglassie @codyborders take a look at the changes we made here, i don't think the methods were doing what the spec was asking but we fixed them!